### DISCUSSION TITLE
Uncaught TypeError: editor.chain().focus().setImage is not a function with vue3 composition api

### DISCUSSION THREAD

**Discussion by mydidefix** [ueberdosis/tiptap#4129] [General Questions]
Hi guys, I have implemented everything under Vue3 in Nuxt3 with Compostion API and everything is running great except for the image extension. Does anyone have a solution to the problem.
When calling the example script under Vue3 composition api 
with <script setup>example code from Tutorial</script>
comes the error! 
Uncaught TypeError: editor.chain().focus().setImage is not a function 

With kind regards
Dieter

**Comment by Muhammetmyrat** [comment]
const editor = ref(
  useEditor({
    content: "<p>I’m running Tiptap with Vue.js. 🎉</p>",
    extensions: [StarterKit],
  })
);

function setParagraph() {
   editor.value.chain().focus().setParagraph().run();
}

**Reply by Muhammetmyrat** [reply]
useEditor write into ref()

**Comment by claide** [comment]
You forgot to add the `value` when calling the editor.

Change this 
`editor.chain().focus().setImage`

to this
`editor.value.chain().focus().setImage`
